.kokoro/release/stage.sh

```diff
@@ -29,6 +29,7 @@ create_settings_xml_file "settings.xml"
 mvn clean install deploy -B \
   --settings ${MAVEN_SETTINGS_FILE} \
   -DskipTests=true \
+  -Dclirr.skip \
```

Comment:
I'm not familiar with this tool, if it's temporary should there be a TODO comment on when to bring it back? If it's permanent should there be a comment explaining why it's broken and we don't need it?

Reply:
The PR has a description with a linked new issue as to why it's here and how to bring it back. I'll add a TODO too.